### PR TITLE
Adding Curio as a GraphQL user

### PR DESCRIPTION
**Pull request by stevewillard**

### CODE REVIEW

**Comment by facebook-github-bot-7** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-4** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

File: site/users/index.html.js [hunk 1]
```diff
@@ -59,6 +59,11 @@ var logos = [
     img: 'meteor.png',
     link: 'https://medium.com/apollo-stack/snappier-uis-with-apollo-client-graphql-bffab0b52b79#.fr92aru2y'
   },
+  {
+    name: 'Curio',
+    img: 'https://s3.amazonaws.com/curio-public/Curio_wordmark_Gradient1.jpg',
```
Comment (leebyron) [inline site/users/index.html.js]: could you add the image to the directory on the site? We want to make sure the site can still load locally when people are editing!

Comment (stevewillard) [inline site/users/index.html.js]: Sure thing.

**Comment by leebyron** [comment]
Could you also remove the margins from the image? That's what's causing this logo to appear smaller and more spaced out than the others, and I wouldn't want your logo to be smaller!

**Comment by stevewillard** [comment]
Good call. Thanks!

**Comment by leebyron** [comment]
Sweet!

**Pull request merged**